README.md

```diff
@@ -105,11 +105,11 @@ export UPGRADE_STRATEGY=composer
 ```
 #### Composer update strategy
 Composer install|update uses two strategies.
-By default composer updates only lock file.
+By default Composer updates lock file with packages being downloaded.
 
-* To enable update composer with packages:
+* If you don't want to run the install step after updating the `composer.lock` file, use:
```

Comment:
`run the install step`
is a bit vague, can we make this more concrete?

Reply:
I took this wording from official composer documentation : https://getcomposer.org/doc/03-cli.md

```
--no-install: Does not run the install step after updating the composer.lock file.
```

Reply:
We need to clarify what it means for us.